azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/header/InfoHeaderView.kt

```diff
@@ -80,6 +80,8 @@ internal class InfoHeaderView : ConstraintLayout {
 
         viewLifecycleOwner.lifecycleScope.launch {
             infoHeaderViewModel.getCallDurationManager()?.timerTickStateFlow?.collect {
+                val formattedTime = infoHeaderViewModel.getFormattedElapsedDuration()
+                timerText.contentDescription = "$formattedTime"
```

Comment:
nit: no need for string interpolation (or the var), just assign the string directly.